projects/azure/README.md

```diff
@@ -22,3 +22,43 @@ Run job:
 ```
 ci-operator --config ci-operator/config/openshift/openshift-azure/master.json --namespace=namespace-name --git-ref=openshift/openshift-azure@master --template ci-operator/templates/cluster-launch-e2e-azure.yaml --secret-dir $(pwd)/cluster/test-deploy/azure/
 ```
+
+# Secret rotation
+
+OSA jobs are using `Web API App` credentials on Azure to run jobs. If for some reason you need to rotate secret, follow this process:
+
+1. Go to `Azure Active Directory` -> `App Registrations` -> `ci-operator-jobs` -> `Settings` -> `Keys`
+2. Delete old key, and create new one.
+3. Create secret example file:
+
+```
+export AZURE_CLIENT_ID=<web app id>
+export AZURE_CLIENT_SECRET=<new key>
+export AZURE_TENANT_ID=<tenant id>
+export AZURE_SUBSCRIPTION_ID=<subscription id>
+```
+
+4. Create a secret
+
+```
+oc create secret generic cluster-secrets-azure --from-file=cluster/test-deploy/azure/secret -o yaml --dry-run | oc apply -n ci -f -	
+```
+
+5. (Optional, if you dont have access to CI namespace)
+
+```
+oc apply secret generic cluster-secrets-azure-temp --from-file=cluster/test-deploy/azure/secret -o yaml --dry-run | oc apply -n azure -f -
+```
+
+and ask somebody, who has access to execute:
+
+```
+oc get secret cluster-secrets-azure-temp --export -n azure -o yaml | sed 's/cluster-secrets-azure-temp/cluster-secrets-azure/g' | oc apply -f - -n ci
```

Comment:
Why not use the same name in the azure namespace and avoid the sed?

Reply:
because we use it already, but the secret type is diff.
options here:
1. We need to refactor purge to use file-based secret so all secrets are same (templates forcing us to use file based)
2. rename purge secret to something else. 
3. do nothing :)

Reply:
@mjudeikis what's forcing us to use the secret of the form `export AZURE_FOO=bar; export AZURE_BAZ=bash` ?  Is that standard across ci-operator?  It's pretty lame.  Using the purge style secret you can do
`export AZURE_CLIENT_ID=$(</path/to/secret/azure_client_id)` in the ci-operator bash script instead.

Reply:
So the thing is that operator works on basis "give me a folder and I will give you a secret". So we have to have file-based secret there with "export FOO=bar" inside the file.

Are we fine to move to same secret for `azure-purge`?